src/nethsec/utils/__init__.py

```diff
@@ -323,3 +323,50 @@ def get_group_macs(uci, group):
     for u in uci.get('objects', group, 'user', list=True, default=[]):
         macs = macs + get_user_macs(uci, u)
     return macs
+
+
+def validation_errors(errors):
+    '''
+    Generate a validation error for the APIs from an array.
+
+    Arguments:
+      - errors -- An array of array errors
+
+    Each array element is an array composed by 3 parameters:
+      - the name of the parameter
+      - the validation error reason
+      - the parameter original value that caused the error
+
+    Returns:
+      - A validation error object
+    '''
+    verrors = []
+    for e in errors:
+        verrors.append({"parameter": e[0], "message": e[1].strip().replace(" ", "_").lower(), "value": e[2]})
+
+    return {"validation": {"errors": verrors}}
+
+def validation_error(parameter, message="", value=""):
```

Comment:
We can just push a single value array to `validation_errors` so we don't duplicate the code

Reply:
I agree